### ISSUE TITLE
We need to stop listing the Google Form as a way of contributing

### ISSUE THREAD

**Issue by bernhold** [betterscientificsoftware/bssw.io#1579]
The notifications only go to the owner of the form, which does not provide sufficient redundancy/reliability for this to be a primary submission mechanism.

**Comment by rinkug** [comment]
Discussed Apt 13; 

* keep the form and rehost it? Owners can change ownership.
* Folks voted to remove the google form. 
- we discussed putting email address directly (bssw-info mailing list address) instead of the google form and this email mailing list  has Rinku, David, Lois, Jim W, Hai-Ah. We usually get fellowship queries on it.
- We discussed changing the email address where the https://bssw.io/contact inputs could go (currently goes bssw-info)
- We also get spam on bssw-info, so putting many receivers to receive this is useless. We cannot find a reliable way to filter the spam 
- We also have an alias for this mailing list (info@bssw) that is used for mailing list.

Action:

1. Remove the below from https://bssw.io/pages/how-to-contribute-content-to-better-scientific-software. **@rinkug  to remove this blurb.**

<img width="606" alt="Screenshot 2023-04-13 at 10 23 23 AM" src="https://user-images.githubusercontent.com/768870/231836880-3d1f2549-11d8-4e85-8f91-d0887e38d544.png">

2. We are going to ask Parallactic to add an items to dropbox. see issue #1618
